airflow/providers/jira/.latest-doc-only-change.txt

```diff
@@ -1 +1 @@
-6c3a67d4fccafe4ab6cd9ec8c7bacf2677f17038
+06acf40a4337759797f666d5bb27a5a393b74fed
```

Comment:
Why no change log for Jira?
We need to release also 3.1.0 for Jira https://github.com/apache/airflow/pull/25930 so users will get deprecations warnings before we remove the provider in next phase

Reply:
Changelog was there already but yeah Provider.yaml was not updeted. Fixed.